src/main/java/org/spongepowered/api/item/inventory/custom/CustomInventory.java

```diff
@@ -54,7 +55,7 @@ static Builder builder() {
          * @param name The title
          * @return fluent pattern
          */
-        Builder name(Translation name);
+        Builder name(String name);
```

Comment:
InventoryBasic wants a String. I am not sure how to convert the Text without loss.
It gets recreated using:
`this.hasCustomName() ? new ChatComponentText(this.getName()) : new ChatComponentTranslation(this.getName(), new Object[0]);`